### PR TITLE
Update the issue templates

### PR DESCRIPTION
**Pull request by rbsec**
Update the issue templates to be cleaner by turning the questions into headings and adding comments to guide the user.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_cheatsheet_proposal.md [hunk 1]
```diff
@@ -1,17 +1,32 @@
 ---
-name: new_cheatsheet_proposal
-about: Used to create a proposal to add a new cheat sheet to the project
+name: New Cheat Sheet Proposal
+about: Used to create a proposal to add a new cheat sheet to the project.
 title: New cheat sheet proposal
```
Comment (mackowski) [inline .github/ISSUE_TEMPLATE/new_cheatsheet_proposal.md]: Maybe include here placeholder for CS name, similar to what we have for update. E.g.:
`New cheat sheet proposal: [PUT_TARGET_CS_NAME_HERE]`

Comment (rbsec) [reply .github/ISSUE_TEMPLATE/new_cheatsheet_proposal.md]: Good call, fixed in 9384911ba05a3719895acc308e528176af92c5cf.

File: .github/ISSUE_TEMPLATE/new_cheatsheet_proposal.md [hunk 2]
```diff
@@ -1,17 +1,32 @@
 ---
-name: new_cheatsheet_proposal
-about: Used to create a proposal to add a new cheat sheet to the project
-title: New cheat sheet proposal
+name: New Cheat Sheet Proposal
+about: Used to create a proposal to add a new cheat sheet to the project.
+title: New cheat sheet proposal: [PUT_TARGET_CS_NAME_HERE]
```
Comment (ThunderSon) [inline .github/ISSUE_TEMPLATE/new_cheatsheet_proposal.md]: Let's change cheat sheet to CS. Makes it smaller and is actually understandable.

Comment (rbsec) [reply .github/ISSUE_TEMPLATE/new_cheatsheet_proposal.md]: Done in a754728.

**Comment by rbsec** [comment]
@ThunderSon I've made the change you suggested above - I'll merge now.

**Pull request merged**